*** bundle/pydiction/plugin/python_pydiction.vim~	2012-08-23 14:14:08.000000000 +0800
--- bundle/pydiction/plugin/python_pydiction.vim	2012-08-23 14:50:11.000000000 +0800
***************
*** 59,71 ****
  
  
  " Make the Tab key do python code completion:
! inoremap <silent> <buffer> <Tab> 
           \<C-R>=<SID>SetVals()<CR>
           \<C-R>=<SID>TabComplete('down')<CR>
           \<C-R>=<SID>RestoreVals()<CR>
  
  " Make Shift+Tab do python code completion in the reverse direction:
! inoremap <silent> <buffer> <S-Tab> 
           \<C-R>=<SID>SetVals()<CR>
           \<C-R>=<SID>TabComplete('up')<CR>
           \<C-R>=<SID>RestoreVals()<CR>
--- 59,71 ----
  
  
  " Make the Tab key do python code completion:
! inoremap <silent> <buffer> <C-J> 
           \<C-R>=<SID>SetVals()<CR>
           \<C-R>=<SID>TabComplete('down')<CR>
           \<C-R>=<SID>RestoreVals()<CR>
  
  " Make Shift+Tab do python code completion in the reverse direction:
! inoremap <silent> <buffer> <C-R> 
           \<C-R>=<SID>SetVals()<CR>
           \<C-R>=<SID>TabComplete('up')<CR>
           \<C-R>=<SID>RestoreVals()<CR>
